src/quotes/quotes.js

```diff
@@ -0,0 +1,148 @@
+import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../constants/roles';
+import { BRAG, CONFESS } from '../constants/roleActions';
+
+export const PAST = 'past';
+export const FUTURE = 'future';
+export const PREFIX = 'prefix';
+
+// ORDER HERE MATTERS, ALWAYS ADD AT THE END
+const PREFIX_LEAF = {
+  [BRAG]: {
+    [PAST]: [
+      'Yesterday I finally',
+      'I have managed and',
+      'At last, I',
+      'Happy to state, I have',
+      'I had a great day and',
+      'Finally it worked! I have'
+    ],
+    [FUTURE]: [
+      'Now I can',
+      'At last I can',
+      'This unlocked me to have',
+      'Today I will,',
+      'Will continue to',
+      'Plan to further'
+    ]
+  },
+  [CONFESS]: {
+    [FUTURE]: [
+      'My current approach is to',
+      'Now I will',
+      'Today, I will retry to',
+      'I will try hard to',
+      'Today, plan to continue to',
+      'Hopefully today, '
+    ],
+    [PAST]: [
+      'Sadly yesterday I have not',
+      'I failed in having',
+      'Failed to have',
+      'I stand correct as I',
+      'Yesterday I failed and have not',
+      "Sadly I haven't "
+    ]
+  }
+};
+
+const DEVOPS_LEAF = {
```

Comment:
I think it's good idea to move each role to separate file. It will be easier for outsiders to contribute that way.

Reply:
as above